Changes

```diff
@@ -1,3 +1,11 @@
+Working version
+-----------------------------
+
+- #9752: Revised handling of calling conventions for external C functions.
+   Provide a more precise description of the types of unboxed arguments,
+   so that the ARM64 iOS/macOS calling conventions can be honored.
+   (Xavier Leroy, review by Mark Shinwell and Github user @EduardoRFS)
+
```

Comment:
If you wish to keep the Changes log accurate, you should mention the (backport of the) original ARM64/macOS pull request.